### PR TITLE
DevNexus24 Post

### PR DESCRIPTION
**Pull request by gljrobinson**

### CODE REVIEW

**Review by yeekangc — DISMISSED** [review]

File: posts/2024-03-06-DevNexus24.adoc [hunk 1]
```diff
@@ -0,0 +1,150 @@
+---
+layout: post
+title: "Our guide to Open Source Cloud Native Java at DevNexus 2024"
```
Comment (dmuelle) [inline posts/2024-03-06-DevNexus24.adoc]:
```suggestion
title: "Our guide to open source cloud native Java at DevNexus 2024"
```

File: posts/2024-03-06-DevNexus24.adoc [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+# Do NOT change the categories section
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/GraceJansen
+author_github: https://github.com/GraceJansen
+seo-title: Our guide to Open Source Cloud Native Java at DevNexus 2024 - OpenLiberty.io
```
Comment (dmuelle) [inline posts/2024-03-06-DevNexus24.adoc]:
```suggestion
seo-title: Our guide to open source cloud native Java at DevNexus 2024 - OpenLiberty.io
```

File: posts/2024-03-06-DevNexus24.adoc [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+seo-description: A useful guide to the sessions and activities going on at DevNexus 2024 focused on open source, cloud native Java technologies, including Open Liberty, MicroProfile, Jakarta EE, CRIU, and more!
```
Comment (dmuelle) [inline posts/2024-03-06-DevNexus24.adoc]:
```suggestion
seo-description: A useful guide to the sessions and activities at DevNexus 2024 that focus on open source, cloud native Java technologies, including Open Liberty, MicroProfile, Jakarta EE, CRIU, and more!
```

File: posts/2024-03-06-DevNexus24.adoc [hunk 4]
```diff
@@ -0,0 +1,150 @@
...
+blog_description: "A useful guide to the sessions and activities going on at DevNexus 2024 focused on open source, cloud native Java technologies, including Open Liberty, MicroProfile, Jakarta EE, CRIU, and more!"
```
Comment (dmuelle) [inline posts/2024-03-06-DevNexus24.adoc]:
```suggestion
blog_description: "A useful guide to the sessions and activities at DevNexus 2024 that focus on open source, cloud native Java technologies, including Open Liberty, MicroProfile, Jakarta EE, CRIU, and more!"
```

File: posts/2024-03-06-DevNexus24.adoc [hunk 5]
```diff
@@ -0,0 +1,150 @@
...
+open-graph-image: https://openliberty.io/img/twitter_card.jpg
+open-graph-image-alt: Open Liberty Logo
+---
+= Our guide to Open Source Cloud Native Java at DevNexus 2024
```
Comment (dmuelle) [inline posts/2024-03-06-DevNexus24.adoc]:
```suggestion
= Our guide to open source cloud native Java at DevNexus 2024
```

File: posts/2024-03-06-DevNexus24.adoc [hunk 6]
```diff
@@ -0,0 +1,150 @@
...
+Grace Jansen <https://github.com/GraceJansen>
+:imagesdir: /
+:url-prefix:
+:url-about: /
+//Blank line here is necessary before starting the body of the post.
+
+== What is DevNexus?
+link:https://devnexus.com/[DevNexus] is the longest-running and largest Java Conference in the USA. It’s organised by a vibrant Java community who are passionate about enabling education for all and empowering the Java developer community. Last year's event brought more than 1400 developers together, coming from all over the world, and more than 90 speakers shared their knowledge on exciting, trending topics including Software architecture, core Java, Kubernetes, and Security.
```
Comment (dmuelle) [inline posts/2024-03-06-DevNexus24.adoc]:
```suggestion
link:https://devnexus.com/[DevNexus] is the longest-running and largest Java Conference in the US. It’s organised by a vibrant Java community who are passionate about enabling education for all and empowering the Java developer community. Last year's event brought more than 1400 developers together from all over the world, and more than 90 speakers shared their knowledge on exciting, trending topics like software architecture, core Java, Kubernetes, and security. 
```

File: posts/2024-03-06-DevNexus24.adoc [hunk 7]
```diff
@@ -0,0 +1,150 @@
...
+
+== What is DevNexus?
+link:https://devnexus.com/[DevNexus] is the longest-running and largest Java Conference in the USA. It’s organised by a vibrant Java community who are passionate about enabling education for all and empowering the Java developer community. Last year's event brought more than 1400 developers together, coming from all over the world, and more than 90 speakers shared their knowledge on exciting, trending topics including Software architecture, core Java, Kubernetes, and Security. 
+
+image::/img/blog/DevNexus2024.jpg[DevNexus 2024 site,width=50%,align="left"]
+
+You can see more of the 2023 highlights in this promo link:https://www.youtube.com/watch?v=x2pYHLx70rQ[video]. For those of you who are keen eyed, you might even spot our very own link:https://github.com/OpenLiberty/space-rover-mission#readme[Liberty Space Rover]!
+
+
+This year the event is being held in downtown Atlanta at the Georgia World Congress Center from April 9th-11th and tickets are already sold out! It's sure to be an awesome event with 145 international speakers taking the stage.
```
Comment (dmuelle) [inline posts/2024-03-06-DevNexus24.adoc]:
```suggestion
This year the event is being held in downtown Atlanta at the Georgia World Congress Center from April 9th-11th and tickets are already sold out! It's sure to be an awesome event, with 145 international speakers taking the stage.
```

File: posts/2024-03-06-DevNexus24.adoc [hunk 8]
```diff
@@ -0,0 +1,150 @@
...
+So, if you're coming to DevNexus, be sure to pop by our Open Liberty booth in the breaks and say hi. You may even get some cool Open Liberty swag!
+
+
+== Which session should I attend?
+
+The great part about attending a conference like DevNexus is the huge array of topics, sessions and speakers presenting throughout the event. You have the chance to learn about everything from agile, architecture and AI, through to cloud infrastructure, cloud technologies and core Java/Jakarta EE. But with over 140 different sessions to choose from, and up to 13 sessions all occurring at the same time, how do you choose which sessions to attend?!
+
+Deciding which sessions to see can be a real challenge! But to help make this easier, we’ve put together this guide of the sessions available on each day focused on open source, cloud-native Java technologies, including: Open Liberty, MicroProfile, Jakarta EE, and more. 
+
+Note: There's also a dedicated Jakarta EE track this year, so make sure to check out the sessions in this.
```
Comment (dmuelle) [inline posts/2024-03-06-DevNexus24.adoc]:
```suggestion
There's also a dedicated Jakarta EE track this year, so make sure to check out those sessions.
```

File: posts/2024-03-06-DevNexus24.adoc [hunk 9]
```diff
@@ -0,0 +1,150 @@
...
+
+So, if you're coming to DevNexus, be sure to pop by our Open Liberty booth in the breaks and say hi. You may even get some cool Open Liberty swag!
+
+
+== Which session should I attend?
+
+The great part about attending a conference like DevNexus is the huge array of topics, sessions and speakers presenting throughout the event. You have the chance to learn about everything from agile, architecture and AI, through to cloud infrastructure, cloud technologies and core Java/Jakarta EE. But with over 140 different sessions to choose from, and up to 13 sessions all occurring at the same time, how do you choose which sessions to attend?!
+
+Deciding which sessions to see can be a real challenge! But to help make this easier, we’ve put together this guide of the sessions available on each day focused on open source, cloud-native Java technologies, including: Open Liberty, MicroProfile, Jakarta EE, and more. 
+
```
Comment (dmuelle) [inline posts/2024-03-06-DevNexus24.adoc]:
```suggestion
```

File: posts/2024-03-06-DevNexus24.adoc [hunk 10]
```diff
@@ -0,0 +1,150 @@
...
+Note: There's also a dedicated Jakarta EE track this year, so make sure to check out the sessions in this.
+
+image::/img/blog/DevNexusJakartaEETrack.jpg[DevNexus Jakarta EE track,width=50%,align="left"]
+
+
+=== Tuesday 9th April 2024
+
+Workshop: link:https://devnexus.com/presentations/the-ultimate-one-day-java-workshop-deploying-ai-enhanced-applications-with-security-scalability[The Ultimate One-Day Java Workshop: Deploying AI-Enhanced Applications with Security & Scalability]
+
+This all day workshop is sure to be a great hands-on learning experience, with 5 internationally renowned speakers leading it. This workshop aims to bridge the gap between traditional Java application deployment and modern cloud-based solutions, offering participants a deep dive into the latest technologies, methodologies, and best practices, including the exciting realm of AI.
```
Comment (dmuelle) [inline posts/2024-03-06-DevNexus24.adoc]:
```suggestion
This all-day workshop, led by 5 internationally renowned speakers, is sure to be a great hands-on learning experience. It bridges the gap between traditional Java application deployment and modern cloud-based solutions, offering participants a deep dive into the latest technologies, methodologies, and best practices, including the exciting realm of AI. 
```

File: posts/2024-03-06-DevNexus24.adoc [hunk 11]
```diff
@@ -0,0 +1,150 @@
...
+This all day workshop is sure to be a great hands-on learning experience, with 5 internationally renowned speakers leading it. This workshop aims to bridge the gap between traditional Java application deployment and modern cloud-based solutions, offering participants a deep dive into the latest technologies, methodologies, and best practices, including the exciting realm of AI. 
+
+
+=== Wednesday 10th April 2024
+
+Time: 10:00-11:00
+
+Talk: link:https://devnexus.com/presentations/jakarta-ee-11-performance-and-developer-productivity[Jakarta EE 11 - Performance and Developer Productivity]
+
+Jakarta EE guru Ivar Grimstad takes us on a journey of understanding how the namespace change from `javax.*` to `jakarta.*` in Jakarta EE 9 impacted the entire Java ecosystem, the cool new features introduced in Jakarta EE 10 and the renewed focus on performance and developer productivity introduced in Jakarta EE 11.
```
Comment (dmuelle) [inline posts/2024-03-06-DevNexus24.adoc]:
```suggestion
Jakarta EE guru Ivar Grimstad takes us on a journey to understand how the namespace change from `javax.*` to `jakarta.*` in Jakarta EE 9 impacted the entire Java ecosystem, the cool new features introduced in Jakarta EE 10, and the renewed focus on performance and developer productivity introduced in Jakarta EE 11.
```

File: posts/2024-03-06-DevNexus24.adoc [hunk 12]
```diff
@@ -0,0 +1,150 @@
...
+Talk: link:https://devnexus.com/presentations/jakarta-ee-the-future-of-enterprise-application-behind-the-myths[Jakarta EE! The future of enterprise application behind the myths.]
+
+Concerned about moving your applications from JEE 5 & 7 to Jakarta EE 10? Join this session by Alberto Salazar to understand the potential pitfalls and put away the concerns of updating and evolving your enterprise applications. 
+
+
+Time: 17:00-18:00
+
+Talk: link:https://devnexus.com/presentations/the-next-frontier-in-open-source-java-compilers-just-in-time-compilation-as-a-service[The Next Frontier in Open Source Java Compilers: Just-In-Time Compilation as a Service]
+
+In this session, Rich Hagarty will dig into JITServer technology, showing the challenges of implementation, detailing its strengths and weaknesses and illustrating its performance characteristics. For those interested in cloud native deployment, he will also show how it can be deployed in containers, demonstrate its advantages compared to a traditional JIT compilation technique and offer practical recommendations about when to use this technology.
```
Comment (dmuelle) [inline posts/2024-03-06-DevNexus24.adoc]:
```suggestion
In this session, Rich Hagarty will dig into JITServer technology, showing the challenges of implementation, detailing its strengths and weaknesses, and illustrating its performance characteristics. For those interested in cloud native deployment, he will also show how to deploy it in containers, demonstrate its advantages compared to a traditional JIT compilation technique, and offer practical recommendations about when to use this technology. 
```

File: posts/2024-03-06-DevNexus24.adoc [hunk 13]
```diff
@@ -0,0 +1,150 @@
...
+In this session, Rich Hagarty will dig into JITServer technology, showing the challenges of implementation, detailing its strengths and weaknesses and illustrating its performance characteristics. For those interested in cloud native deployment, he will also show how it can be deployed in containers, demonstrate its advantages compared to a traditional JIT compilation technique and offer practical recommendations about when to use this technology. 
+
+
+=== Thursday 12th April 2024
+
+Time: 10:00-11:00
+
+Talk: link:https://devnexus.com/presentations/making-the-best-of-jakarta-ee-kubernetes-websphere-and-the-cloud-with-ai[Making the best of Jakarta EE, Kubernetes, WebSphere and the cloud with AI]
+
+This hands-on session will show you how to enable deployments of Jakarta EE applications using Open Liberty or WebSphere Liberty to the cloud with AI. With two Java champions presenting this session, it's sure to be a great session!
```
Comment (dmuelle) [inline posts/2024-03-06-DevNexus24.adoc]:
```suggestion
This hands-on session will show you how to enable deployments of Jakarta EE applications using Open Liberty or WebSphere Liberty to the cloud with AI. With two Java champions presenting, it's sure to be a great session!
```

File: posts/2024-03-06-DevNexus24.adoc [hunk 14]
```diff
@@ -0,0 +1,150 @@
...
+Talk: link:https://devnexus.com/presentations/cache-me-if-you-can-speed-up-your-jvm-with-project-valhalla[Cache Me If You Can: Speed Up Your JVM With Project Valhalla]
+
+OpenJDK’s Project Valhalla is set to revolutionize how the JVM handles data objects in memory. This talk offers an exploration of Valhalla's objectives, progress, and its potential impact on the Java ecosystem with examples throughout. Join this session to discover how Project Valhalla is on the verge of reshaping the landscape of Java, paving the way for a more efficient and high-performing future. 
+
+
+Time: 13:30-14:30
+
+Talk: link:https://devnexus.com/presentations/databaseless-data-processing-with-eclipsestore-and-websphere-liberty-instanton[Databaseless Data Processing with EclipseStore and WebSphere Liberty InstantOn]
+
+Join Markus Kett and Java Champion Emily Jiang in this session to learn how the combinaton of Liberty InstantOn and EclipseStore enables Java applications to achieve the 3 most critical factors for cloud-native applicatons making use of AI, ML, VR and big data in the cloud: high performance, low data storage costs, and simple, fast and cost-efficient implementation and maintenance.
```
Comment (dmuelle) [inline posts/2024-03-06-DevNexus24.adoc]:
```suggestion
Join Markus Kett and Java Champion Emily Jiang in this session to learn how the combinaton of Liberty InstantOn and EclipseStore enables Java applications to achieve the 3 most critical factors for cloud-native applicatons making use of AI, ML, VR, and big data in the cloud: high performance, low data storage costs, and simple, fast and cost-efficient implementation and maintenance.
```

File: posts/2024-03-06-DevNexus24.adoc [hunk 15]
```diff
@@ -0,0 +1,150 @@
...
+Time: 16:00-17:00
+
+Talk: link:https://devnexus.com/presentations/jakarta-ee-11-going-beyond-the-era-of-java-ee[Jakarta EE 11: Going Beyond the Era of Java EE]
+
+Java guru Michael Redlich presents this session providing a history of JavaEE/Jakarta EE and a review of new and updated specifications with code examples. 
+
+
+== IBM Booth:
+
+As well as the numerous sessions during the conference and our dedicated Open Liberty booth, the Open Liberty team will also be manning the IBM booth in the exhibition hall. Stop by to catch our inbooth theatre with a series of SmartTalks, grab some awesome swag, and have a chat about all things OSS Java. We’d love to see you there!
```
Comment (dmuelle) [inline posts/2024-03-06-DevNexus24.adoc]:
```suggestion
As well as the numerous sessions during the conference and our dedicated Open Liberty booth, the Open Liberty team will also be manning the IBM booth in the exhibition hall. Stop by to catch our in-booth theatre with a series of SmartTalks, grab some awesome swag, and have a chat about all things OSS Java. We’d love to see you there!
```

**Review by dmuelle — CHANGES_REQUESTED** [review]
ID review- a few minor suggestions

**Review by dmuelle — APPROVED** [review]

**Pull request merged**